onnxruntime/test/providers/cpu/nn/pool_op_test.cc

```diff
@@ -967,6 +967,31 @@ TEST(PoolTest, AveragePool_19_dilation_2d) {
   test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider, kAclExecutionProvider, kOpenVINOExecutionProvider});
 }
 
+TEST(PoolTest, AveragePool_19_ceil_count_include_pad_1d) {
+  // TODO: Unskip when fixed #41968513
```

Comment:
is #41968513 a valid issue?

Reply:
I think so. I checked there are other 14 test cases having this comment. I guess they have it somewhere.